medic/cht-core
# Docker helper script for CHT 4.x

# Description

Adds one liner support for app developers to easily start up a new CHT instance(s).

See [nascent docs](https://docs.communityhealthtoolkit.org/apps/guides/hosting/4.x/app-developer/#cht-docker-helper-for-4x).

medic/cht-core#7852

To test:
* check out this branch
* cd into `scripts/docker-helper-4.x`
* run `./cht-docker-compose.sh` OR `./cht-docker-compose.sh EXISTING-PROJECT.env` OR `./cht-docker-compose.sh EXISTING-PROJECT.env down` OR  `./cht-docker-compose.sh EXISTING-PROJECT.env destroy`
* data is stored in `~/.medic/`

# Code review checklist


# Compose URLs:





* CHT Core: CHT_CORE_COMPOSE_URL
* CouchDB Single: COUCH_SINGLE_COMPOSE_URL
* CouchDB Cluster: COUCH_CLUSTER_COMPOSE_URL
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.
